hbw/hbw.gemspec

```diff
@@ -15,9 +15,16 @@ Gem::Specification.new do |s|
 
   s.files = Dir['{app,config,db,lib}/**/*']
 
+  s.add_dependency 'actionmailer', '~> 5.2.4'
+  s.add_dependency 'actionpack', '~> 5.2.4'
+  s.add_dependency 'actionview', '~> 5.2.4'
+  s.add_dependency 'activejob', '~> 5.2.4'
+  s.add_dependency 'activemodel', '~> 5.2.4'
+  s.add_dependency 'activerecord', '~> 5.2.4'
+  s.add_dependency 'activesupport', '~> 5.2.4'
   s.add_dependency 'coffee-rails', '~> 4.2'
   s.add_dependency 'faraday'
   s.add_dependency 'faraday-detailed_logger'
   s.add_dependency 'faraday_middleware'
-  s.add_dependency 'rails', '~> 5.2.4'
+  s.add_dependency 'railties', '~> 5.2.4'
```

Comment:
If we really need sprockets-rails then why isnt presented here?

Reply:
We need sprockets-rails for apitome gem.